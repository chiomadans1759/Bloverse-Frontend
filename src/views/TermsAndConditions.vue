<template>
    <main class="terms">
        <div class="terms-container">
            <h1>Terms And Conditions</h1>
            <p>LAST UPDATED 19 DECEMBER, 2018</p>
            <div class="terms-texts">
                <h1>Welcome to Bloverse!</h1>
                <p>Thanks for using our products and services. By using our Services, you are agreeing to these terms.
                    These ‘’Terms’’ are a contract between you and Bloverse. Please read them carefully;
                    <br/><br/>These Terms can be changed at any time. We would send you a notification each time there is an addition or change in these Terms.
                </p>
                <h5>Rights &amp; responsibilities on Content</h5>
                <p> You own the rights to the content you create and post on Bloverse.<br/><br/> You’re responsible for the 
                    content you post. This means you assume all risks related to it, including someone else’s claims 
                    relating to intellectual property or other legal rights.<br/><br/> You’re welcome to share your published 
                    content on Bloverse to other social platforms or accounts of yours. <br/><br/>By posting content to Bloverse, 
                    you have given us the license to use your content. We may also use your content to 
                    promote Bloverse. We will never sell your content to third parties without your explicit permission.                
                </p>
                <h5>Our content and services</h5>
                <p>Do not interfere with, or disrupt, the access of any user, host, or network, including sending a virus, 
                    overloading, flooding, spamming, mail-bombing the Services, or by scripting the creation of content or 
                    accounts in such a manner as to interfere with or create an undue burden on the Services. <br/><br/> Scraping the 
                    Services is totally prohibited, but Crawling the Services is allowed if done in accordance with the provisions 
                    of our robots.txt file
                </p>
                <h5>No children</h5>
                <p>Bloverse is only for people 13 years old and over. By using Bloverse, you affirm that you are over 13. If we learn someone 
                    under 13 is using Bloverse, we’ll terminate their account.
                </p>
                <h5>Incorporated rules and policies</h5>
                <p>By using the Services, you agree to let Bloverse collect and use information as detailed in our Privacy Policy. <br/><br/>To enable a 
                    functioning community, we have House Rules for commenters, House rules for content providers.<br/><br/> By using Bloverse, you agree to 
                    follow these Rules and Policies. If you don’t, we may remove your content, or suspend or delete your account.
                </p>
                <h5>Other Terms</h5>
                <p><b>Disclaimer of warranty <br/></b>No implied warranty of merchantability, fitness for a particular purpose, availability, security, title or non-infringement.
                     You use our services at your own risk and discretion. That means they don’t come with any warranty. <br/><br/><b>Responsibilities for comments <br/></b>Kindly note that 
                     any opinion or comment being share on your post isn’t of Bloverse but of commenters. <br/><br/><b>Entire agreement. <br/></b>These Terms (including any document 
                     incorporated by reference into them) are the whole agreement between Bloverse and you concerning the Services.
                </p>
            </div>
        </div>
    </main>
</template>

<script>
export default {
    
}
</script>
terms-texts
<style scoped>
    .terms{
        background-color: #F5F5F5;
        display:flex;
        justify-content: center;
        align-items: center;
        font-family: Montserrat;
    }
    .terms-container{
        width:50%;
        margin-bottom:7rem;
    }
    .terms-container h1{
        font-weight:700;
        letter-spacing: 2px;
    }
    .terms-container > p{
        font-size:.6rem;
        color:#096DD9;
        font-weight:600;
        letter-spacing: 2px;
        padding:1rem 0 2rem 0;
    }
    .terms-texts{
        background-color: #fff; 
        padding:1rem 2.5rem 2rem 2.5rem;
    }
    .terms-texts h1{ 
        font-weight:700;
        padding-bottom:1rem;
        letter-spacing:2px;
        line-height: 2;
    }
    .terms-texts h5{ 
        font-weight:700;
        padding:1rem 0;
        letter-spacing:2px;
        line-height: 2;
    }
    .terms-texts p{ 
        line-height: 2;
    }
    @media screen and (max-width: 800px) {
    .terms-container{
        width:90%; 
    }  
    }
</style>
