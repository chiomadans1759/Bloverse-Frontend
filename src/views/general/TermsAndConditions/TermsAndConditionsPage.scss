@import 'sass-rem';

.terms-nav .nav-link {
  padding-right: 0 !important;
  padding-left: 0 !important;
}
.terms {
  font-family: Montserrat, Regular;
  color: #2d2d2d;
  font-weight: 500;
}

.navbar-nav {
  font-size: rem(16px);
  font-weight: 700;
}

nav {
  margin: 0;
  padding: 0;
}

.sub-content {
  word-spacing: rem(2px);
  font-size: rem(14px);
  color: #767676;
}

.sub-content > h6 {
  color: #565656;
}

.last-update {
  color: #565656;
}
.date {
  font-weight: 700;
}

.terms-title > h1 {
  font-size: rem(26px);
}

.terms-title > p {
  font-size: rem(14px);
}


.contest-text {
  font-size: 1rem;
  color: #767676;
}

.contest-header {
  font-size: 1rem;
}

.contest-title {
  font-size: 1rem;
  font-weight: 900;
}

.contest-content {
  padding-top: 1.5rem;
}

.navigation-bar {
  background-color: #ffffff;
}

.terms-content {
  background-color: #ffffff;
}

.nav-link {
  font-size: 1rem;
  color: #565656 !important;
  border: none !important;
  height: 3.75rem;
}

.nav-link.active {
  font-weight: 900 !important;
  background: transparent !important;
  border-bottom: 0.0625rem solid #004ded !important;
}

.photocontest {
  padding: 1.5rem;
}



@media screen and (max-width: 800px) {
  .terms-container{
    width:100% !important;
  }

  nav {
    display: none;
  }

  .terms-container{
    width:100% !important;
  }

  .terms-title {
    padding-left: rem(20px);
  }
}

@media screen and (max-width: 768px) and (min-width: 600){
  .terms-container{
    width:80% !important;
  }
}

@media screen and (min-width: 1024px) {
  .terms-container{
    width:50% !important;
  }
}
