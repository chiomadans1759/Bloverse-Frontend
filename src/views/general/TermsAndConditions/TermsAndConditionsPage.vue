<template>
    <div class="terms d-flex justify-content-center align-items-center">
        <div class="terms-container w-50 mb-5">
          <div class="terms-title">
            <h1 class="font-weight-bold mb-2">Terms And Conditions</h1>
            <p class="last-update text-capitalize">Last Updated <span class="date">19 December, 2018</span></p>
          </div>
          <div class="terms-content border mt-4 container p-0">
            <div class="navigation-bar">
              <ul class="nav nav-tabs mb-3 justify-content-center" id="pills-tab" role="tablist">
                <li class="nav-item">
                  <a class="nav-link active d-flex align-items-center" id="pills-general-tab" data-toggle="pill" href="#general-tab" role="tab" aria-controls="pills-general" aria-selected="true">General</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link d-flex align-items-center" id="pills-contest-tab" data-toggle="pill" href="#photocontest-tab" role="tab" aria-controls="pills-contest" aria-selected="false">Photo Contest</a>
                </li>
              </ul>
            </div>
            <div class="tab-content" id="terms-tab">
              <div class="tab-pane fade general show active" id="general-tab" role="tabpanel" aria-labelledby="general-tab">
                <div class="bg-white mt-4 pt-0 mb-5">
                  <div class="general p-4 pb-5">
                    <div class="sub-content">
                      <h6 class="font-weight-bold pb-4">Rights &amp; responsibilities on Content</h6>
                      <p class="pb-4">You own the rights to the content you create and post on Bloverse.</p>
                      <p class="pb-4">
                        You’re responsible for the content you post.
                        This means you assume all risks related to it,
                        including someone else’s claims relating to intellectual property
                        or other legal rights
                      </p>
                      <p class="pb-4">
                        You’re welcome to share your published content on Bloverse
                        to other social platforms or accounts of yours.
                      </p>
                      <p class="pb-4">
                        By posting content to Bloverse, you have given us the license to use your content. We may also use your content to promote Bloverse. 
                        We will never sell your content to third parties without your explicit permission.
                      </p>
                    </div>
                    <div class="sub-content">
                      <h6 class="font-weight-bold pb-4">Our content and services</h6>
                      <p class="pb-4"> Do not interfere with, or disrupt, the access of any user,
                        host, or network, including sending a virus, overloading,
                        flooding, spamming, mail-bombing the Services, or by scripting
                        the creation of content or accounts in such a manner as to interfere with
                        or create an undue burden on the Services.</p>
                      <p class="pb-4">
                          Scraping the Services is totally
                          prohibited, but Crawling the Services is allowed if done in accordance with the provisions of our robots.txt file
                      </p>
                    </div>
                    <div class="sub-content">
                      <h6 class="font-weight-bold pb-4">No children</h6>
                      <p class="pb-4">
                        Bloverse is only for people 13 years old and over.
                        By using Bloverse, you affirm that you are over 13.
                        If we learn someone under 13 is using Bloverse, we’ll terminate their account.
                      </p>
                    </div>
                    <div class="sub-content">
                      <h6 class="font-weight-bold pb-4">Incorporated rules and policies</h6>
                      <p class="pb-4">
                        By using the Services, you agree to let Bloverse collect and
                        use information as detailed in our Privacy Policy.
                      </p>
                      <p class="pb-4">
                        To enable a functioning community, we have House Rules for commenters, 
                        House rules for content providers
                      </p>
                      <p class="pb-4">
                        By using Bloverse, you agree to follow these Rules and Policies. If you don’t, 
                        we may remove your content, or suspend or delete your account.
                      </p>
                    </div>
                    <div class="sub-content">
                      <h6 class="font-weight-bold pb-4">Other Terms</h6>
                      <p class="pb-4">
                        Bloverse is only for people 13 years old and over.
                        By using Bloverse, you affirm that you are over 13.
                        If we learn someone under 13 is using Bloverse, we’ll terminate their account.
                      </p>
                    </div>
                    <div class="sub-content">
                      <h6 class="font-weight-bold pb-4">Incorporated rules and policies</h6>
                      <p class="pb-4">
                        Disclaimer of warranty
                      </p>
                      <p class="pb-4">
                        No implied warranty of merchantability,
                        fitness for a particular purpose, availability,
                        security, title or non-infringement. You use our services at your own
                        risk and discretion. That means they don’t come with any warranty.
                      </p>
                      <p class="pb-4">
                        Responsibilities for comments
                      </p>
                      <p class="pb-4">
                        Kindly note that any opinion or comment being share on your post
                        isn’t of Bloverse but of commenters.
                      </p>
                      <p class="pb-4">
                        Entire agreement.
                      </p>
                      <p class="pb-4">
                        These Terms (including any document incorporated by reference into them)
                        are the whole agreement between Bloverse and you concerning the Services.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
              <div class="tab-pane fade photocontest" id="photocontest-tab" role="tabpanel" aria-labelledby="photocontest-tab">
                <div class="contest-content">
                  <p class="contest-header">
                    DO NOT PARTICIPATE IN THE CONTEST IF YOU DO NOT FALL WITHIN THE ELIGIBILITY REQUIREMENTS AS SET FORTH BELOW.
                    <br /><br />
                    By entering this Contest, entrants agree to abide by the Official Contest Rules &amp; Regulations.
                    <br /><br />
                    NO PURCHASE NECESSARY. The BLOVERSE ‘Story Behind The Photo’ is sponsored by Bloverse.
                    The Contest begins at [Time ] on [date] and ends at [time] on [date]
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">Eligibility</p>
                  <br />
                  <p class="contest-text">
                    To enter and be eligible to win, an entrant must be a legal resident of the 20 countries mentioned above
                    who at the time of entry has reached the age of majority or older in the province/territory in 
                    which he/she resides and the entrant must not be, or be domiciled with, a Contest Party or an immediate 
                    family member thereof. Contest Parties means employees, representatives and agents of (1) Sponsor, 
                    (2) Sponsor’s (i) affiliated and related companies and (ii) advertising or promotion agencies and (3) 
                    entities involved in the development, production, administration, or fulfillment of the Contest. 
                    In these Official Rules, “immediate family member” means the entrant’s mother, father, brother, 
                    sister, son, daughter or legal or common-law spouse.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">How to enter the contest</p>
                  <br />
                  <p class="contest-text">
                    No purchase necessary. Purchase is not required to enter the Contest. 
                    There is one (1) method of entry to enter the Contest: Online Entry: 
                    Visit (the “Contest Website”) during the Contest Period. Next, click on the “Sign Up” 
                    button and enter your name, email address, phone number, your country and mailing address 
                    and only if the following statements are true, confirm that (i) you have reached the age 
                    of majority in your province or territory of residence and (ii) you have read and agree with 
                    the Official Rules. Limit of one (1) entry per person subject to the eligibility requirements set 
                    out in these Official Rules, one entry entitles a person to one chance to win. All entries must 
                    be received on or before [time] on [date]. Winners will be emailed with the email address 
                    provided in the sign up form. Winners will have to provide paypal details within 7 days so 
                    they can receive the cash transfer. Limitations: There is a limit of one (1) First Place Prize 
                    per person. There is a limit of one (1) Second Place Prize per person. limit of one (1) Third 
                    Place Prize per person. A person is ineligible to be declared the winner of any further Prize 
                    if he or she has reached the limit for the given Prize type, as set out in this Rule. If it is 
                    discovered by Bloverse (using any evidence or other information made available to or otherwise 
                    discovered by Bloverse) that any person has attempted to: (i) obtain more than one (1) entry 
                    (regardless of the method of entry); and/or (ii) used multiple names, multiple identities and/or 
                    any other means not in keeping with the Bloverse’s interpretation of the letter and/or spirit of 
                    these Rules to enter or otherwise participate in or to disrupt this Contest; then he/she may be disqualified 
                    from the Contest in the sole and absolute discretion Bloverse. Bloverse is not responsible for, and accepts 
                    no liability whatsoever in relation to, any late, lost, misdirected, delayed, illegible, inaccurate or 
                    incomplete Entries (all of which are void). Any entry may be rejected, denied and disqualified if, in 
                    the sole and absolute discretion of the Sponsor the entry is not submitted and received in accordance 
                    with these Rules.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">Prices and Odds</p>
                  <br />
                  <p class="contest-text">
                    There will be a total of three winners. There is 1 First Place Prize to be won ($200),
                    1 Second Place Prize to be won ($150) and 1 Third Place Prize ($100). Prizes must be accepted 
                    as awarded without substitution, and are not (1) transferable, (2) or for resale. The Bloverse 
                    reserves the right, in its sole and absolute discretion, in the event that a Prize cannot be 
                    awarded as described for any reason, to substitute a Prize of equal or greater value without liability.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">Winner Notification Process</p>
                  <br />
                  <p class="contest-text">
                    Within seven (7) business days of the Draw Date, Bloverse will make five (5) attempts to contact 
                    the selected entrant by email (using the information provided at the time of entry). If the 
                    selected entrant cannot be contacted within five (5) attempts or seven (7) business days 
                    of the Draw Date (whichever occurs first), or there is a return of any notification as 
                    undeliverable, then that selected entrant will be disqualified. In the event of a 
                    disqualification, or in the event that Bloverse otherwise determines, in its sole 
                    and absolute discretion, that a selected entrant is ineligible under the terms of 
                    these Official Rules, Bloverse reserves the right, but is not required, to randomly 
                    select an alternate eligible entrant from among the remaining eligible Contest entries.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">General</p>
                  <br />
                  <p class="contest-text">
                    By entering the Contest, entrants agree to abide by the Official Contest Rules 
                    and the decisions of the Sponsor with respect to all aspects of the Contest, which are final.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">Terms of Entry</p>
                  <br />
                  <p class="contest-text">
                    By entering this Contest, in consideration for the chance to win a Prize, 
                    each entrant authorizes Bloverse to use, his/her name, city of residence, stories, 
                    photographs and (or) other image for advertising purposes, without any remuneration 
                    whatsoever, unless prohibited by law.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">Release</p>
                  <br />
                  <p class="contest-text">
                    You release the Releases from any liability in connection with a Prize or this Contest. 
                    The Releasees will not be responsible for late, lost, illegible, falsified, damaged, 
                    misdirected, mutilated, garbled or incomplete entries or Prize claims and all such 
                    entries/Prize claims are void. All entries become the property of Bloverse upon receipt 
                    and none will be returned. The Releasees will not be liable for any failure of the 
                    Contest Website, do not assume any responsibility for incorrect or inaccurate capture 
                    of entry information, technical malfunctions, human or technical error, printing errors, 
                    lost, delayed or garbled data or transmissions, omission, interruption, deletion, defect 
                    or failures of any telephone or computer lines or networks, computer equipment, software 
                    or any combination thereof. Entry material/data that have been tampered with or altered 
                    are void. The Releasees shall not be liable to Prize winners or any other person for 
                    failure to supply any Prize or any part thereof by reason of any acts of God, any actions, 
                    regulations, orders, or requests by any governmental entity, equipment failure, terrorist 
                    acts, war, fire, unusually severe weather, embargo, labour dispute or strike, labour or material 
                    shortage, transportation interruption of any kind, or any other cause beyond the reasonable 
                    control of the Releases.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">Contest Changes</p>
                  <br />
                  <p class="contest-text">
                    Bloverse reserves the right, with the consent of the Régie des alcools, des courses et des jeux 
                    (the “Régie”), to terminate or suspend this Contest or to amend the Official Contest Rules &amp; Regulations 
                    at any time and in any way, without prior notice. Without limiting the foregoing, if, 
                    for any reason, the Contest is not capable of running as originally planned, Bloverse reserves the right, 
                    with the consent of the Régie, to cancel the Contest. Bloverse shall not be held responsible for any problems, 
                    errors or negligence that may arise or occur in connection with the Contest.
                  </p>
                </div>
                <div class="contest-content">
                  <p class="contest-title">Consent to use personal information</p>
                  <br />
                  <p class="contest-text">
                    When you voluntarily choose to participate in the Contest, personal information including, 
                    but not limited to your: first name, last name, mailing address, city, province/territory, 
                    postal code, telephone and email (collectively “Registrant Information”) will be collected 
                    from you by Bloverse. By entering this Contest and providing this information, you consent 
                    and agree to Bloverses’ collection and use of the Registrant Information to administer this 
                    Contest and in accordance with Sponsor’s privacy policy. Further, if the Photo uploaded by 
                    you contains personal information about you or about a child for whom you are the parent, 
                    guardian or other authorized legal representative, you consent to the use of the Photo. Please 
                    note that personal information provided as part of this Contest may be collected in, transferred 
                    to and processed and stored in jurisdictions outside of the 20 countries. Such information will 
                    be subject to the general laws applicable within those jurisdictions, including, without limitation, 
                    possible access by regulatory authorities.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
    </div>
</template>

<script>
export default {}
</script>

<style scoped lang="scss">
  @import './TermsAndConditionsPage';
</style>
