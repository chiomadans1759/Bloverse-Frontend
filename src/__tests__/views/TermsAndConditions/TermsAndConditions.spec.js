import { mount, RouterLinkStub } from '@vue/test-utils';
import TermsAndConditions from '../../../views/general/TermsAndConditions/TermsAndConditionsPage.vue';

describe('Terms And Conditions', () => {
  it('should render correctly', () => {
    const wrapper = mount(TermsAndConditions, {
      stubs: {
        RouterLink: RouterLinkStub
      }
    });

    expect(wrapper.element).toMatchSnapshot();
  })
})
