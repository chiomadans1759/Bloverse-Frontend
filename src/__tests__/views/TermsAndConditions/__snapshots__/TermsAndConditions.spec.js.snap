// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Terms And Conditions should render correctly 1`] = `
<div
  class="terms d-flex justify-content-center align-items-center"
>
  <div
    class="terms-container w-50 mb-5"
  >
    <div
      class="terms-title"
    >
      <h1
        class="font-weight-bold mb-2"
      >
        Terms And Conditions
      </h1>
       
      <p
        class="last-update text-capitalize"
      >
        Last Updated 
        <span
          class="date"
        >
          19 December, 2018
        </span>
      </p>
    </div>
     
    <div
      class="terms-content bg-white mt-4 pt-0 mb-5 border"
    >
      <div
        class="terms-nav border-bottom"
      >
        <nav
          class="navbar navbar-expand-lg navbar-light bg-white justify-content-center"
        >
          <div>
            <ul
              class="navbar-nav"
            >
              <li
                class="nav-item active mr-4 p-2"
              >
                <a
                  class="nav-link"
                >
                   General 
                  <span
                    class="sr-only"
                  >
                    (current)
                  </span>
                </a>
              </li>
               
              <li
                class="nav-item p-2"
              >
                <a
                  class="nav-link"
                >
                  Photo Contest
                </a>
              </li>
            </ul>
          </div>
        </nav>
      </div>
       
      <div
        class="general p-4 pb-5"
      >
        <div
          class="sub-content"
        >
          <h6
            class="font-weight-bold pb-4"
          >
            Rights & responsibilities on Content
          </h6>
           
          <p
            class="pb-4"
          >
            You own the rights to the content you create and post on Bloverse.
          </p>
           
          <p
            class="pb-4"
          >
            
                  You’re responsible for the content you post.
                  This means you assume all risks related to it,
                  including someone else’s claims relating to intellectual property
                  or other legal rights
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  You’re welcome to share your published content on Bloverse
                  to other social platforms or accounts of yours.
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  By posting content to Bloverse, you have given us the license to use your content. We may also use your content to promote Bloverse. 
                  We will never sell your content to third parties without your explicit permission.
                
          </p>
        </div>
         
        <div
          class="sub-content"
        >
          <h6
            class="font-weight-bold pb-4"
          >
            Our content and services
          </h6>
           
          <p
            class="pb-4"
          >
             Do not interfere with, or disrupt, the access of any user,
                  host, or network, including sending a virus, overloading,
                  flooding, spamming, mail-bombing the Services, or by scripting
                  the creation of content or accounts in such a manner as to interfere with
                  or create an undue burden on the Services.
          </p>
           
          <p
            class="pb-4"
          >
            
                    Scraping the Services is totally
                    prohibited, but Crawling the Services is allowed if done in accordance with the provisions of our robots.txt file
                
          </p>
        </div>
         
        <div
          class="sub-content"
        >
          <h6
            class="font-weight-bold pb-4"
          >
            No children
          </h6>
           
          <p
            class="pb-4"
          >
            
                  Bloverse is only for people 13 years old and over.
                  By using Bloverse, you affirm that you are over 13.
                  If we learn someone under 13 is using Bloverse, we’ll terminate their account.
                
          </p>
        </div>
         
        <div
          class="sub-content"
        >
          <h6
            class="font-weight-bold pb-4"
          >
            Incorporated rules and policies
          </h6>
           
          <p
            class="pb-4"
          >
            
                  By using the Services, you agree to let Bloverse collect and
                  use information as detailed in our Privacy Policy.
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  To enable a functioning community, we have House Rules for commenters, 
                  House rules for content providers
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  By using Bloverse, you agree to follow these Rules and Policies. If you don’t, 
                  we may remove your content, or suspend or delete your account.
                
          </p>
        </div>
         
        <div
          class="sub-content"
        >
          <h6
            class="font-weight-bold pb-4"
          >
            Other Terms
          </h6>
           
          <p
            class="pb-4"
          >
            
                  Bloverse is only for people 13 years old and over.
                  By using Bloverse, you affirm that you are over 13.
                  If we learn someone under 13 is using Bloverse, we’ll terminate their account.
                
          </p>
        </div>
         
        <div
          class="sub-content"
        >
          <h6
            class="font-weight-bold pb-4"
          >
            Incorporated rules and policies
          </h6>
           
          <p
            class="pb-4"
          >
            
                  Disclaimer of warranty
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  No implied warranty of merchantability,
                  fitness for a particular purpose, availability,
                  security, title or non-infringement. You use our services at your own
                  risk and discretion. That means they don’t come with any warranty.
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  Responsibilities for comments
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  Kindly note that any opinion or comment being share on your post
                  isn’t of Bloverse but of commenters.
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  Entire agreement.
                
          </p>
           
          <p
            class="pb-4"
          >
            
                  These Terms (including any document incorporated by reference into them)
                  are the whole agreement between Bloverse and you concerning the Services.
                
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
